recipe/meta.yaml

```diff
@@ -142,6 +143,10 @@ test:
         - test -f $PREFIX/lib/libopencv_freetype${SHLIB_EXT}     # [unix]
         - python -c "import cv2; assert 'Unknown' not in cv2.videoio_registry.getBackendName(cv2.CAP_V4L)"  # [linux]
         - python -c "import cv2, re; assert re.search('Lapack:\s+YES', cv2.getBuildInformation())"
+        - mkdir -p cmake_build_test && pushd cmake_build_test
+        - cmake -G "${CMAKE_GENERATOR}" -D "OpenCV_DIR=${PREFIX}/cmake" "${RECIPE_DIR}"
```

Comment:
there is a way to specify a test file that should be copied into the testing stage. Not sure if that is what is giving you trouble. You are really being thorough :D.